aes/aesni/src/aes128.rs

```diff
@@ -19,8 +19,9 @@ pub struct Aes128 {
 impl Aes128 {
     #[inline(always)]
     pub(crate) fn encrypt8(&self, mut blocks: [__m128i; 8]) -> [__m128i; 8] {
-        let keys = self.encrypt_keys;
-        unsafe {
+        #[inline]
+        #[target_feature(enable = "aes")]
```

Comment:
Please add a full list of features, which you can find in `src/target_checks.rs`. It should improve performance a bit further.

Reply:
Per https://doc.rust-lang.org/reference/attributes/codegen.html "aes" implies "sse2", and the only use of SSSE3 is in counter mode for pshufb. So I'm not sure if it makes any difference, though I could change it if you prefer for consistency.

Reply:
>"aes" implies "sse2"

Ah, good to know.